client/state/sites/reducer.js

```diff
@@ -133,7 +133,7 @@ export const items = withSchemaValidation( sitesSchema, ( state = null, action )
 			let nextSite = site;
 
 			return reduce(
-				[ 'blog_public', 'wpcom_coming_soon', 'site_icon' ],
+				[ 'blog_public', 'wpcom_public_coming_soon', 'site_icon' ],
```

Comment:
Is it possible to leave the old version in play concurrently?

In #45606 I'm putting things that access these states behind `config.isEnabled('coming-soon-v2')` checks so having both floating around should be fine.

Reply:
> Is it possible to leave the old version in play concurrently?

Good question. I'll look into this. I haven't yet put too much deep though into that.

> behind config.isEnabled('coming-soon-v2')

I've been using the `editing-toolkit/coming-soon` flag. Could we use that to keep things consistent??

Reply:
I caved. Switched to `coming-soon-v2` in this PR